README.md

```diff
@@ -0,0 +1,49 @@
+# global-workflow
+Global Superstructure/Workflow currently supporting the Finite-Volume on a Cubed-Sphere Global Forecast System (FV3GFS)
+
+The global-workflow depends on the following prerequisities to be available on the system:
+
+* workload management platform / scheduler - LSF or SLURM
+* workflow manager - ROCOTO (https://github.com/christopherwharrop/rocoto)
+* modules - NCEPLIBS (various), esmf v8.0.0bs48, hdf5, intel/ips v18, impi v18, wgrib2, netcdf v4.7.0, hpss, gempak (see module files under /modulefiles for additional details)
+* manage_externals - A utility from ESMCI to checkout external dependencies. Manage_externals can be obtained at the following address and should be in the users PATH: https://github.com/ESMCI/manage_externals
```

Comment:
You might want to add a blurb on what `manage_externals` is replacing (`checkout.sh`).
I also suggest either removing `checkout.sh` or put in big bold letters that support for `checkout.sh` is being deprecated, future PR's will be denied if an inconsistencies are found between `checkout.sh` and `Externals.cfg` and `checkout.sh` will be removed on a not-so-distant future date.